Sources/StreamChat/Controllers/MessageController/MessageController.swift

```diff
@@ -300,18 +300,22 @@ public extension ChatMessageController {
             return
         }
 
-        let lastLocalMessageId: () -> MessageId? = {
```

Comment:
Any reason why we got rid of this? I guess `lastFetchedMessageId` should be the right source of truth, right?

Reply:
Yea exactly, if we don't have it, it means we are fetching the first page